iosMath/render/MTMathUILabel.m

```diff
@@ -73,11 +103,19 @@ - (void)setFontSize:(CGFloat)fontSize
     self.font = font;
 }
 
+#if TARGET_OS_IPHONE
 - (void)setContentInsets:(UIEdgeInsets)contentInsets
+#else
+- (void)setContentInsets:(NSEdgeInsets)contentInsets
+#endif
 {
     _contentInsets = contentInsets;
     [self invalidateIntrinsicContentSize];
+#if TARGET_OS_IPHONE
     [self setNeedsLayout];
+#else
+    [self setNeedsLayout:YES];
```

Comment:
Just define a method
```objective-c
- (void)setNeedsLayout:(BOOL) val {
    [self setNeedsLayout]
}
```
for the iPhone side. Then you can avoid all these `#if #else`.